[2024-10-04T16:52:27.987+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-10-04T16:52:28.030+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: minio_api_call_dag.call_api_task manual__2024-10-04T16:52:26.502073+00:00 [queued]>
[2024-10-04T16:52:28.055+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: minio_api_call_dag.call_api_task manual__2024-10-04T16:52:26.502073+00:00 [queued]>
[2024-10-04T16:52:28.056+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 2
[2024-10-04T16:52:28.100+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): call_api_task> on 2024-10-04 16:52:26.502073+00:00
[2024-10-04T16:52:28.118+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:62 DeprecationWarning: This process (pid=192) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-10-04T16:52:28.120+0000] {standard_task_runner.py:64} INFO - Started process 200 to run task
[2024-10-04T16:52:28.117+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'minio_api_call_dag', 'call_api_task', 'manual__2024-10-04T16:52:26.502073+00:00', '--job-id', '46', '--raw', '--subdir', 'DAGS_FOLDER/dag_to_api.py', '--cfg-path', '/tmp/tmpq6payak9']
[2024-10-04T16:52:28.123+0000] {standard_task_runner.py:91} INFO - Job 46: Subtask call_api_task
[2024-10-04T16:52:28.201+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/settings.py:195 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2024-10-04T16:52:28.397+0000] {task_command.py:426} INFO - Running <TaskInstance: minio_api_call_dag.call_api_task manual__2024-10-04T16:52:26.502073+00:00 [running]> on host 3fed06331d8d
[2024-10-04T16:52:28.686+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_EMAIL='admin' AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='minio_api_call_dag' AIRFLOW_CTX_TASK_ID='call_api_task' AIRFLOW_CTX_EXECUTION_DATE='2024-10-04T16:52:26.502073+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-10-04T16:52:26.502073+00:00'
[2024-10-04T16:52:28.689+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-10-04T16:52:28.968+0000] {logging_mixin.py:188} INFO - File C:\Users\ADMIN\Downloads\Documents\setup_***\dags\api_data.json đã được tạo thành công trên máy local.
[2024-10-04T16:52:28.970+0000] {python.py:237} INFO - Done. Returned value was: None
[2024-10-04T16:52:28.972+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-10-04T16:52:28.999+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=minio_api_call_dag, task_id=call_api_task, run_id=manual__2024-10-04T16:52:26.502073+00:00, execution_date=20241004T165226, start_date=20241004T165228, end_date=20241004T165228
[2024-10-04T16:52:29.076+0000] {local_task_job_runner.py:243} INFO - Task exited with return code 0
[2024-10-04T16:52:29.118+0000] {taskinstance.py:3503} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-10-04T16:52:29.128+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
